soccer/Referee.hpp

```diff
@@ -2,119 +2,32 @@
 
 #include <protobuf/LogFrame.pb.h>
 #include <protobuf/referee.pb.h>
-#include <stdint.h>
+#include <boost/asio.hpp>
+#include <boost/config.hpp>
+#include <cstdint>
+#include "RefereeEnums.h"
 
-#include <QMutex>
-#include <QThread>
-#include <QTime>
 #include <Utils.hpp>
+#include <future>
```

Comment:
What is this being used for?

Reply:
woops it used to be used but I removed it